src/main/java/mate/academy/carsharing/dto/payment/PaymentRequestDto.java

```diff
@@ -0,0 +1,12 @@
+package mate.academy.carsharing.dto.payment;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Positive;
+import lombok.Data;
+
+@Data
+public class PaymentRequestDto {
+    @NotBlank
```

Comment:
NotBlank is used for String
```suggestion
    @NotNull
```